tracing/plugin/otlp.go

```diff
@@ -105,7 +106,7 @@ func newExporter(ctx context.Context, cfg *OTLPConfig) (*otlptrace.Exporter, err
 	if cfg.Protocol == "http/protobuf" || cfg.Protocol == "" {
 		u, err := url.Parse(cfg.Endpoint)
 		if err != nil {
-			return nil, fmt.Errorf("OpenTelemetry endpoint %q is invalid: %w", cfg.Endpoint, err)
+			return nil, fmt.Errorf("OpenTelemetry endpoint %q is invalid: %w", cfg.Endpoint, errdefs.ErrInvalidArgument)
```

Comment:
Why discard err?

Reply:
I was thinking categorizing the err based on the error type,  so that later unit test can match the category. 
A second thought is probably I should add err back.